tests/unit/src/services/BigNumber.spec.js

```diff
@@ -0,0 +1,12 @@
+import BigNumber from '@/services/BigNumber'
+
+describe('should operate with big numbers', () => {
```

Comment:
My preference to test that is with a [scoped jest mock](https://jestjs.io/docs/en/manual-mocks#mocking-node-modules) of `big.js` library and check that the different operations of our wrapper call the correct method of `big.js`